activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -461,7 +462,6 @@ def initialize_type_map(m) # :nodoc:
           m.register_type 'ltree', OID::SpecializedString.new(:ltree)
 
           # FIXME: why are we keeping these types as strings?
-          m.alias_type 'interval', 'varchar'
```

Comment:
Can you point me on some example how to do that (deprecation warning and some kind of switch to enable this feature immediately)?